src/features/filter.ts

```diff
@@ -1,12 +1,26 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Status } from '../types/Status';
 
 const initialState = {
   query: '',
-  status: 'all',
+  status: 'all' as Status,
```

Comment:
Ensure that the `Status` type includes the 'all' value, as it is used as the initial state for the status.